inst/extdata/packages/wallace.yaml

```diff
@@ -1,6 +1,6 @@
 name : wallace
 title: "A Modular Platform for Reproducible Modeling of Species Niches and Distributions"
-version: 2.0.3
+version: 2.0.4
```

Comment:
@jamiemkass & @adamlilith , as planned you can see here that submission help keeping other packages up-to-date.